test/mustachio/renderer_test.dart

```diff
@@ -403,11 +403,13 @@ void main() {
     var foo = Foo();
     expect(
         () => renderFoo(foo, fooTemplate),
-        throwsA(const TypeMatcher<MustachioResolutionError>().having(
-            (e) => e.message,
-            'message',
-            contains('Failed to resolve s2 as a property on any types in the '
-                'context chain: Foo'))));
+        throwsA(const TypeMatcher<MustachioResolutionError>()
+            .having((e) => e.message, 'message', contains('''
+line 1, column 8 of ${fooTemplateFile.path}: Failed to resolve 's2' as a property on any types in the context chain: Foo
```

Comment:
now these errors are starting to look better than the ones given by mustache, awesome!

Reply:
:D :D :D